src/cmd/deploy.go

```diff
@@ -14,8 +14,6 @@ import (
 	"github.com/spf13/viper"
 )
 
-var integrationUrl string
```

Comment:
Using a `var` here is not necessary and the env var binding does not update this, causing the bug where setting `OPSLEVEL_INTEGRATION_URL` does nothing.